akka-docs/rst/java/distributed-data.rst

```diff
@@ -503,6 +503,18 @@ Note that you should be prepared to receive ``WriteFailure`` as reply to an ``Up
 durable entry if the data could not be stored for some reason. When enabling ``write-behind-interval``
 such errors will only be logged and ``UpdateSuccess`` will still be the reply to the ``Update``.
 
+There is one important caveat when it comes pruning of :ref:`crdt_garbage_java` for durable data.
+If and old data entry that was never pruned is injected and merged with existing data after 
+that the pruning markers have been removed the value will not be correct. The time-to-live
+of the markers is defined by configuration 
+``akka.cluster.distributed-data.durable.remove-pruning-marker-after`` and is in the magnitude of days.
+This would be possible if replica with durable data didn't participate in the pruning
+(e.g. it was shutdown) and later started after this time. A durable replica should not 
+be stopped for longer time than this duration and if it is joining again after this
```

Comment:
So, if the actual replicator was stopped and restarted on a node, but the cluster node never left the cluster, or the node was partitioned for the entire time? I wonder if this can be made more clear by being a bit more explicit.